### PR TITLE
Releases for vsphere

### PR DESCRIPTION
**Pull request by njuettner**
Towards: https://github.com/giantswarm/roadmap/issues/3646

### Checklist

- [ ] Roadmap issue created
- [ ] Release uses latest stable Flatcar
- [ ] Release uses latest Kubernetes patch version

### Triggering E2E tests

To trigger the E2E test for each new Release added in this PR, add a comment with the following:

`/run releases-test-suites`

If you want to trigger conformance tests, you can do so by adding a comment similar to the following:

`/run conformance-tests PROVIDER=capa RELEASE_VERSION=29.1.0`

For more details see the [README.md](/README.md#running-tests-against-prs).

### CODE REVIEW

File: vsphere/v27.0.0/release.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  - name: vertical-pod-autoscaler
+    version: 5.2.4
+    dependsOn:
+    - prometheus-operator-crd
+  - name: vertical-pod-autoscaler-crd
+    version: 3.1.0
+  components:
+  - name: cluster-vsphere
+    catalog: cluster-test
+    version: 0.64.0-3f8dd6331f18146f4eac8cd4fae53422d171afba
```
Comment (njuettner) [inline vsphere/v27.0.0/release.yaml:94]: This will change once we have the first release version for `cluster-vpshere`

**Review by glitchcrab — APPROVED** [review]

**Review by anvddriesch — APPROVED** [review]

**Comment by vxav** [comment]
I know nothing about releases so I'll refrain from approving this and I'll sit the next 101 🙂

**Comment by njuettner** [comment]
Standard test is suceeding, upgrade test is expected to fail

image

**Review by vxav — APPROVED** [review]

**Pull request merged**